source/settings/settings-reference.txt

```diff
@@ -147,6 +147,10 @@ You can configure the following settings on the |compass| interface:
      - Remain logged in when using the :manual:`MONGODB-OIDC </core/security-oidc/>` 
        authentication mechanism. 
 
+   * - Use Generative AI
+     - Feature Preview
+     - Enables :ref:`natural language querying <query-natural-language>`.
```

Comment:
It's not currently showing because the feature is currently disabled for users. In the next beta it will start showing up, sorry for the confusion there. This is a separate preference and not tied to the Atlas log in. If it's turned off then the ai features don't show.